includes/RestApi/SiteGenController.php

```diff
@@ -106,4 +115,143 @@ public function generate_sitegen_meta( \WP_REST_Request $request ) {
 			array( 'status' => 404 )
 		);
 	}
+
+	/**
+	 * Get Sitegen Customize sidebar data.
+	 *
+	 * @param \WP_REST_Request $request Request model.
+	 *
+	 * @return array|WP_Error
+	 */
+	public function get_customize_sidebar_data( \WP_REST_Request $request ) {
```

Comment:
I think we can replace this hardcoding with the `nfd-ai-site-gen-colorpalette` option.

Reply:
I replaced only colorPalette and kept the other information needed for default colors/fonts and fonts data required to render fonts section.